libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -35,6 +35,8 @@ void GCS::get_sensor_status_flags(uint32_t &present,
     present = control_sensors_present;
     enabled = control_sensors_enabled;
     health = control_sensors_health;
+
+    return true;
```

Comment:
Why use a bool return if it always returns true?

Reply:
In order to use this function as is, it was necessary to use a bool return due to the following restriction.

"Nullable arguments are only available on a boolean method"
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Scripting/generator/src/main.c#LL828C29-L828C88

Reply:
Probably the same reason here.
https://github.com/ArduPilot/ardupilot/pull/18434/files#diff-352b5b71f52200b52b47c27affe81d92b10aba9de10379b2b9b1f6b2c3f914a3R345